lib/src/builder/parsing/ast_util.dart

```diff
@@ -106,6 +106,15 @@ extension AbstractGetter on ClassOrMixinDeclaration {
           member.returnType?.toSource() == type);
 }
 
+/// An extension that supports APIs that changed from [Identifier] to [Token],
+/// in order to cut down on diffs in the analyzer 5 upgrade (and subsequent
+/// merge conflicts with the null-safety branch.
+///
+/// TODO remove this and inline the [name] member.
+extension NameIdentifierTokenCompat on Token {
+  String get name => lexeme;
+}
+
```

Comment:
This is why there aren't more changes to move to `.lexeme`

Reply:
Yup, exactly. I figure that can happen either in the null safety branch, or after null safety merges